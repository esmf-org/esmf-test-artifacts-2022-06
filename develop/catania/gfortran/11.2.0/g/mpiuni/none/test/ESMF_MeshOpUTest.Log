build time -- 2022-04-20 01:33:04
20220420 014304.373 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220420 014304.373 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220420 014304.373 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220420 014304.373 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220420 014304.373 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220420 014304.373 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220420 014304.373 INFO             PET0 Running with ESMF Version   : v8.3.0b12-72-g1df70b4e76
20220420 014304.373 INFO             PET0 ESMF library build date/time: "Apr 20 2022" "01:31:29"
20220420 014304.373 INFO             PET0 ESMF library build location : /Volumes/esmf/esmf-testing/gfortran_11.2.0_mpiuni_g_develop
20220420 014304.373 INFO             PET0 ESMF_COMM                   : mpiuni
20220420 014304.373 INFO             PET0 ESMF_MOAB                   : enabled
20220420 014304.373 INFO             PET0 ESMF_LAPACK                 : enabled
20220420 014304.373 INFO             PET0 ESMF_NETCDF                 : enabled
20220420 014304.373 INFO             PET0 ESMF_PNETCDF                : disabled
20220420 014304.373 INFO             PET0 ESMF_PIO                    : disabled
20220420 014304.373 INFO             PET0 ESMF_YAMLCPP                : enabled
20220420 014304.373 INFO             PET0  Beginning Test, file ESMF_MeshOpUTest.F90, line 50
20220420 014304.373 INFO             PET0  NUMBER_OF_PROCESSORS           1
20220420 014304.444 INFO             PET0  PASS  exercise difference operation, ESMF_MeshOpUTest.F90, line 60
20220420 014304.444 INFO             PET0  Ending Test, file ESMF_MeshOpUTest.F90, line 62
20220420 014304.444 INFO             PET0 Finalizing ESMF
