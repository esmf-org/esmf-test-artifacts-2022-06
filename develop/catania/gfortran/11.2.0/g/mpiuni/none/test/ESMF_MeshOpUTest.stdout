build time -- 2022-04-28 01:32:35
  Beginning Test, file ESMF_MeshOpUTest.F90, line 50
  NUMBER_OF_PROCESSORS           1
 Starting test3
  PASS  exercise difference operation, ESMF_MeshOpUTest.F90, line 60
  Ending Test, file ESMF_MeshOpUTest.F90, line 62
  PET           0  Test Elapsed Time    70.738000000000028       msec.
